Combined_NN_Model.py

```diff
@@ -1,70 +1,32 @@
 ### import the required modules and functions
-#TODO: Is this the correct Write_Performance()? --> NO, it's not. We should
-# update it with the parameters important for the NN in Utils.py
-# Same for NN_param_typecheck in NN_Utils.py
+
 import subprocess
 import time
 import sys
-
-from pycorenlp import StanfordCoreNLP
-
 from Utils import Write_Performance
 from config import *
-from reddit_parser import Parser
 from ModelEstimation import NNModel
 from transformers import BertTokenizer
 from NN_Utils import *
+from reddit_parser import Parser # Does the parser object need to be adjusted?
 
-# NOTE: Don't forget to set NN=True in defaults.py before running this file
-
-### Define the neural network object
-
-nnmodel=NNModel()
-
-
-### check key hyperparameters for correct data types
-
-nnmodel.NN_param_typecheck()
-
-### Write hyperparameters to file. Performance measures will be written to the
-# same file after analyses are performed
-
-Write_Performance()
-
-### call the parsing function
-
-# CoreNLP
-# create a connection to the CoreNLP server to retrieve sentiment
-# (requires CoreNLP_server.py in the same directory)
-subprocess.Popen(['java -mx6g -cp "*" edu.stanford.nlp.pipeline.StanfordCoreNLPServer --quiet'],
-               shell=True, cwd="./stanford-corenlp-4.0.0")
-time.sleep(5)  # wait for connection to the server to be established
-
+# QUESTION: Does the ID need to show up here in the functions too?
 theparser=Parser()
+
 # Create relevant folders
 theparser.safe_dir_create()
 
 # parse the documents
 theparser.Parse_Rel_RC_Comments()
 
-if Neural_Relevance_Filtering:
-    # Use a transformer-based neural network trained on human ratings to prune the
-    # dataset from irrelevant posts. Path will default to the Human_Ratings folder
-    theparser.Neural_Relevance_Screen()
 
-    # Needs results from Neural_Relevance_Screen
-    theparser.Neural_Relevance_Clean()
+### Define the neural network object
 
-theparser.lang_filtering() # filter non-English posts
+nnmodel=NNModel()
 
-# NOTE: Requires a Neural_Relevance_Screen random sample hand-annotated for accuracy
-if eval_relevance:
-    theparser.eval_relevance()
+### check key hyperparameters for correct data types
 
-### call the function for calculating the percentage of relevant comments
-# NOTE: May work only for full-year sets of dates
-if calculate_perc_rel:
-    theparser.Perc_Rel_RC_Comment()
+nnmodel.NN_param_typecheck()
```

Comment:
I moved NN_param_typecheck() to Utils.py and deleted NN_Utils.py as a file. I'd recommend we implement that simplifying step in this merge, which would mean this function call does not need "nnmodel" at the beginning

Reply:
Can you clarify when you did this step? If it was in the commits placed in the Model-Improvements branch, then I think we should wait until that PR'd into dev.